UniConnectApp/app/src/main/java/com/example/uniconnect/dao/IUniversityDAO.kt

```diff
@@ -11,5 +11,5 @@ interface IUniversityDAO {
      * @return List of universities
      */
     @GET("/search")
-    fun getAllUniversities() : Call<ArrayList<University>>
+    fun getAllUniversities(): Call<ArrayList<University>>
```

Comment:
automatic via kotlin style guide per https://kotlinlang.org/docs/coding-conventions.html#apply-the-style-guide